.github/workflows/ubuntu-ci.yml

```
@@ -0,0 +1,62 @@
+
+name: Shapes Demo Ubuntu CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      os-version:
+        description: 'OS version to run the workflow'
+        required: false
+        default: 'ubuntu-22.04'
+        type: string
+      colcon-args:
+        description: 'Extra arguments for colcon cli'
+        required: false
+        type: string
+      cmake-args:
+        description: 'Extra arguments for cmake cli'
+        required: false
+        type: string
+      ctest-args:
+        description: 'Extra arguments for ctest cli'
+        required: false
+        type: string
+      shapes-demo-branch:
+        description: 'Branch or tag of Shapes Demo repository'
+        type: string
+        required: true
+      fastdds-branch:
+        description: 'Branch or tag of Fast DDS repository'
+        type: string
+        required: true
+      use-ccache:
+        description: 'Use CCache to speed up the build'
+        required: false
+        type: boolean
+        default: false
+
+  pull_request:
+    types:
+      - review_requested
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - '!**/CMakeLists.txt'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  ubuntu-ci:
```

I think that this check can be moved here, so when a PR has conflicts we can see the jobs skipped
```suggestion
  ubuntu-ci:
        if: ${{ (
              !contains(github.event.pull_request.labels.*.name, 'conflicts')
            ) }}
```